tutteli-gradle-spek/src/main/groovy/ch/tutteli/gradle/spek/SpekPlugin.groovy

```diff
@@ -5,13 +5,13 @@ import org.gradle.api.Plugin
 import org.gradle.api.Project
 import org.gradle.api.plugins.UnknownPluginException
 import org.jetbrains.kotlin.gradle.plugin.KotlinPluginWrapper
-import org.junit.platform.gradle.plugin.JUnitPlatformExtension
 
 class SpekPluginExtension {
     String version = '2.0.4'
 }
 
 class SpekPlugin implements Plugin<Project> {
+    static final String JUNIT_PLATFORM_VERSION = "5.5.1"
```

Comment:
remove, only required for jupiter-engine which is not required (see next comment)

Reply:
See other comment. I think jupiter-engine _is_ required.

Reply:
I don't see another way currently, so let's do it this way